model/model.go

```diff
@@ -249,16 +249,22 @@ const (
 	// However, the convert is missed in some scenarios before v2.1.9, so for all those tables prior to TableInfoVersion3, their
 	// charsets / collations will be converted to lower-case while loading from the storage.
 	TableInfoVersion3 = uint16(3)
-	// TableInfoVersion4 indicates that the auto_increment allocator in TiDB has been separated from
+	// TableInfoVersion5 indicates that the auto_increment allocator in TiDB has been separated from
 	// _tidb_rowid allocator. This version is introduced to preserve the compatibility of old tables:
-	// the tables with version < TableInfoVersion4 still use a single allocator for auto_increment and _tidb_rowid.
+	// the tables with version < TableInfoVersion5 still use a single allocator for auto_increment and _tidb_rowid.
 	// Also see https://github.com/pingcap/tidb/issues/982.
-	TableInfoVersion4 = uint16(4)
+	TableInfoVersion5 = uint16(5)
```

Comment:
?? what happened to version 4

Reply:
Version 4 is introduced in https://github.com/pingcap/parser/pull/1170...

It is released to v5.0, so we need another version to identify that the auto IDs are separated.